api/models/dataSource.js

```diff
@@ -0,0 +1,12 @@
+const { DataSource } = require('typeorm');
+
+const AppDataSource = new DataSource({
```

Comment:
```suggestion
const appDataSource = new DataSource({
```
클래스나 enum같은 경우 외에는, 즉 지금 처럼 instance같은 경우에는 대문자로 시작하지 않는 것이 일반적인 컨벤션입니다-!